dietpi-software.html

```diff
@@ -48,12 +48,13 @@
 				<button class="navbar-toggler" data-toggle="collapse" data-target=".navbar-collapse"><svg viewBox="0 0 12 12"><path d="M1 1h10M1 6h10M1 11h10"/></svg></button>
 				<!-- Main navigation -->
 				<div class="navbar-collapse collapse">
-					<a class="nav-link" href="/">Home</a>
-					<a class="nav-link" href="/#features">Features</a>
-					<a class="nav-link" href="/#download">Download</a>
-					<a class="nav-link" href="/#gettingstarted">Getting Started</a>
-					<a class="nav-link" href="/contribute.html">Contribute</a>
-					<a class="nav-link" href="/#testimonials">Testimonials</a>
+					<a class="nav-link" href="#home">Home</a>
+					<a class="nav-link" href="#features">Features</a>
+					<a class="nav-link" href="#download">Download</a>
+					<a class="nav-link" href="#gettingstarted">Getting Started</a>
+					<a class="nav-link" href="#stats">Stats</a>
+					<a class="nav-link" href="contribute.html">Contribute</a>
+					<a class="nav-link" href="#testimonials">Testimonials</a>
```

Comment:
Same as above: Everything needs to stay, to have them navigate to the main index.html, and stats added like contribute, pointing to its own HTML.